[0.309s] Invoking command in '/workspace/src/task01': PYTEST_ADDOPTS=--tb=short --junit-xml=/workspace/build/task01/pytest.xml --junit-prefix=task01 -o cache_dir=/workspace/build/task01/.pytest_cache PYTHONDONTWRITEBYTECODE=1 /usr/bin/python3 -m pytest
[0.681s] [1m============================= test session starts ==============================[0m
[0.681s] platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
[0.685s] cachedir: /workspace/build/task01/.pytest_cache
[0.685s] rootdir: /workspace/src/task01
[0.685s] plugins: ament-copyright-0.14.4, ament-pep257-0.14.4, launch-testing-ros-0.24.1, ament-lint-0.14.4, launch-testing-2.0.4, ament-xmllint-0.14.4, ament-flake8-0.14.4, colcon-core-0.17.0
[0.796s] [1mcollecting ... [0m[1m
[0.796s] collected 3 items                                                              [0m
[0.797s] 
[0.805s] test/test_copyright.py [33ms[0m[32m                                                 [ 33%][0m
[0.992s] test/test_flake8.py [31mF[0m[31m                                                    [ 66%][0m
[1.013s] test/test_pep257.py [32m.[0m[31m                                                    [100%][0m
[1.013s] 
[1.014s] =================================== FAILURES ===================================
[1.014s] [31m[1m_________________________________ test_flake8 __________________________________[0m
[1.014s] [1m[31mtest/test_flake8.py[0m:23: in test_flake8
[1.066s]     [94massert[39;49;00m rc == [94m0[39;49;00m, \
[1.066s] [1m[31mE   AssertionError: Found 1 code style errors / warnings:[0m
[1.066s] [1m[31mE     ./setup.py:1:1: F401 'setuptools.find_packages' imported but unused[0m
[1.066s] [1m[31mE   assert 1 == 0[0m
[1.066s] ----------------------------- Captured stdout call -----------------------------
[1.066s] 
[1.067s] ./setup.py:1:1: F401 'setuptools.find_packages' imported but unused
[1.067s] from setuptools import find_packages, setup
[1.067s] ^
[1.067s] 
[1.067s] 1     F401 'setuptools.find_packages' imported but unused
[1.067s] 
[1.068s] 6 files checked
[1.068s] 1 errors
[1.068s] 
[1.068s] 'F'-type errors: 1
[1.068s] 
[1.068s] Checked files:
[1.068s] 
[1.069s] * ./setup.py
[1.069s] * ./task01/__init__.py
[1.069s] * ./task01/receiver.py
[1.069s] * ./test/test_pep257.py
[1.069s] * ./test/test_flake8.py
[1.069s] * ./test/test_copyright.py
[1.069s] ----------------------------- Captured stderr call -----------------------------
[1.070s] option --max-complexity: please update from optparse string `type=` to argparse callable `type=` -- this will be an error in the future
[1.070s] ------------ generated xml file: /workspace/build/task01/pytest.xml ------------
[1.070s] =========================== short test summary info ============================
[1.070s] FAILED test/test_flake8.py::test_flake8 - AssertionError: Found 1 code style ...
[1.070s] [31m============== [31m[1m1 failed[0m, [32m1 passed[0m, [33m1 skipped[0m, [33m2 warnings[0m[31m in 0.39s[0m[31m ==============[0m
[1.070s] [33m=============================== warnings summary ===============================[0m
[1.070s] test/test_flake8.py::test_flake8
[1.071s] test/test_flake8.py::test_flake8
[1.071s]   Warning: SelectableGroups dict interface is deprecated. Use select.
[1.071s] 
[1.071s] -- Docs: https://docs.pytest.org/en/stable/warnings.html
[1.228s] Invoked command in '/workspace/src/task01' returned '1': PYTEST_ADDOPTS=--tb=short --junit-xml=/workspace/build/task01/pytest.xml --junit-prefix=task01 -o cache_dir=/workspace/build/task01/.pytest_cache PYTHONDONTWRITEBYTECODE=1 /usr/bin/python3 -m pytest
